inst/extdata/volsurface.csv

```diff
@@ -0,0 +1,18 @@
+Code,Type,Tenor,Date,5%,10%,15%,20%,25%,30%,35%,40%,45%,50%,55%,60%,65%,70%,75%,80%,85%,90%,95%,100%,105%,110%,115%,120%,125%,130%,135%,140%,145%,150%,155%,160%,165%,170%,175%,180%,185%,190%,195%,200%,205%,210%,215%,220%,225%,230%,235%,240%,245%,250%,255%
```

Comment:
You may need to update the format of this data (or write another function to transform this data into the format suggested in one of my earlier comments)

Reply:
This data is extracted from a Murex report as an example of vol surface data, it is then massaged in R/utils.R to construct an object following the suggested class structure for VolSurface.